### ISSUE TITLE
Wing it command not working

### ISSUE THREAD

**Issue by ShaiBer** [winglang/wing#2086]
### I tried this

Created the most basic Wing app: 
```ts
bring cloud;

let b = new cloud.Bucket();
```

Then tried running wing it (with and without the file name).

### I expected this:

The wing console to open with a visualization of the contents of my file

### Instead, this happened

Got this error:

```
/Users/shaiber/.volta/tools/image/packages/winglang/lib/node_modules/winglang/dist/commands/run.js:21
    await (0, open_1.default)("wing-console://" + simfile);
                             ^

TypeError: (0 , open_1.default) is not a function
    at Command.run (/Users/shaiber/.volta/tools/image/packages/winglang/lib/node_modules/winglang/dist/commands/run.js:21:30)
    at Command.listener [as _actionHandler] (/Users/shaiber/.volta/tools/image/packages/winglang/lib/node_modules/winglang/node_modules/commander/lib/command.js:482:17)
    at /Users/shaiber/.volta/tools/image/packages/winglang/lib/node_modules/winglang/node_modules/commander/lib/command.js:1283:65
    at Command._chainOrCall (/Users/shaiber/.volta/tools/image/packages/winglang/lib/node_modules/winglang/node_modules/commander/lib/command.js:1177:12)
    at Command._parseCommand (/Users/shaiber/.volta/tools/image/packages/winglang/lib/node_modules/winglang/node_modules/commander/lib/command.js:1283:27)
    at /Users/shaiber/.volta/tools/image/packages/winglang/lib/node_modules/winglang/node_modules/commander/lib/command.js:1081:27
    at Command._chainOrCall (/Users/shaiber/.volta/tools/image/packages/winglang/lib/node_modules/winglang/node_modules/commander/lib/command.js:1177:12)
    at Command._dispatchSubcommand (/Users/shaiber/.volta/tools/image/packages/winglang/lib/node_modules/winglang/node_modules/commander/lib/command.js:1077:23)
    at Command._parseCommand (/Users/shaiber/.volta/tools/image/packages/winglang/lib/node_modules/winglang/node_modules/commander/lib/command.js:1248:19)
    at Command.parse (/Users/shaiber/.volta/tools/image/packages/winglang/lib/node_modules/winglang/node_modules/commander/lib/command.js:909:10)
    at main (/Users/shaiber/.volta/tools/image/packages/winglang/lib/node_modules/winglang/dist/cli.js:61:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.12.1
```

### Is there a workaround?

Yes, starting the wing console like a regular app and then opening the .w file though the GUI

### Component

CLI

### Wing Version

0.11.4

### Wing Console Version

0.2.32 

### Node.js Version

v18.12.1

### Platform(s)

MacOS

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

**Comment by eladb** [comment]
Fixed formatting

**Comment by eladb** [comment]
Suspect: https://github.com/winglang/wing/pull/2083

**Comment by staycoolcall911** [comment]
Fixed by #2089
